source/stdx/allocator/building_blocks/stats_collector.d

```diff
@@ -160,7 +160,7 @@ struct StatsCollector(Allocator, ulong flags = Options.all,
 private:
     import stdx.allocator.internal : Ternary;
 
-    static string define(string type, string[] names...)
+    enum define = (string type, string[] names...)
```

Comment:
interesting, thank you for the approach.

Reply:
Petar Kirov brought it up on the forums, and it has been like a liveraft to me. https://forum.dlang.org/post/akmxdmifdgvqwfhbubej@forum.dlang.org